src/tools/rest-backend/cryptlite/base64.h

```diff
@@ -0,0 +1,198 @@
+/*
```

Comment:
As part of the JWTs (Json Web Tokens). They require the payload to be encrypted with an application secret, so that only the application can read and verify the cookie. I don't think there is another way around this. Using boost directly would bloat the code tremendously, which would result in refactoring into some cryptographic methods, what would lead to some sort of self-written cryptlite library. So I would like to keep it. I don't see a problem in using Boost anyway, because it is seen as part of the standard by most devs as far as I know.